content/docs/cloud/concepts.md

```diff
@@ -3,51 +3,51 @@ title: Concepts
 enableTableOfContents: true
 ---
 
-## User
+## Neon User
 
-Neon User is identified by their email address.
+A Neon user is identified by their email address.
 
-User registers and authenticates in Neon Web UI with their GitHub account. More authentication methods are coming soon.
+A user registers and authenticates in Neon Web UI with their GitHub account. More authentication methods are coming soon.
 
-Once authenticated, Neon User can create and access [Projects](#project) and [query Project data](../tutorials#query-via-ui). They can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
+Once authenticated, a user can create and access [Projects](#project) and [query Project data](../tutorials#query-via-ui). You can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
 
-### API keys
```

Comment:
API keys were subsection of the User section because the API keys are created in scope of the user, not project.
If we're moving the header one level up, we can probably move them lower in the page.
WDYT, @alexandriastech ?

Reply:
To make sure I understand, you want to move "## API Keys" further down on the page?

Another option is to change the title to "## Using API Keys as Neon User" and keep the title under ## Neon User, to avoid confusion

Reply:
I think that API keys should be either a subsection of User or shoud sit further down the page.